src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const newState = { ...state };
+  const returnObjects = [];
+  let container;
+
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'clear') {
+      for (const key in newState) {
+        delete newState[key];
+      }
+      container = Object.assign({}, newState);
+      returnObjects.push(container);
```

Comment:
don't push object to arrayActions in each if you can do it once at the end of the loop